src/libcommonserver/io/iohelper.cpp

```diff
@@ -323,6 +323,67 @@ bool IoHelper::getFileSize(const SyncPath &path, uint64_t &size, IoError &ioErro
     return true;
 }
 
+bool IoHelper::getDirectorySize(const SyncPath &path, uint64_t &size, unsigned int maxDepth, bool &skipedTooDeep,
```

Comment:
As discussed together, I would remove parameter `skippedTooDeep`, set a default value for `maxDepth`and return a specific IoError if maximum depth is reached

Reply:
Done in commits: https://github.com/Infomaniak/desktop-kDrive/pull/100/commits/13ac91508fc1f358d2f9436294069c2f74ce17c0